python/tvm/topi/cuda/conv2d_hwnc_tensorcore.py

```diff
@@ -254,13 +253,8 @@ def schedule_hwnc_tensorcore_cuda(cfg, s, Conv):
     vector_as = cfg["vector_as"].val
     vector_ws = cfg["vector_ws"].val
     split_block_k_nums = cfg["split_block_k_nums"].val
-    fuse_pack = cfg["fuse_pack"].val
 
-    if not fuse_pack:
-        s[packed_data].compute_inline()
-    else:
-        with Target("cuda"):
-            schedule_injective_from_existing(s, packed_data)
+    s[packed_data].compute_inline()
```

Comment:
Please show why narrowing search space

Reply:
having `fuse` and `compute_at` in the search space causes loop itervars to touch buffers a different number of times, which causes errors for AutoTVM's feature extraction, i.e feature extraction will return different size features

Reply:
Does it only happen in `int4` or it is a common problem for both `int8/int4`?
If it is just for `int4`, we should not change it because it may make `int8/float16` slower. alternatively, we can add a single file only for `int4` schedule.

Reply:
this is a problem for both `int4/int8`